dask_jobqueue/oar.py

```diff
@@ -123,6 +163,8 @@ class OARCluster(JobQueueCluster):
         Deprecated: use ``job_extra_directives`` instead. This parameter will be removed in a future version.
     job_extra_directives : list
         List of other OAR options, for example `-t besteffort`. Each option will be prepended with the #OAR prefix.
+    oar_mem_core_property_name : str
+        Syntax for memory per core. If None, warning to users.
```

Comment:
Je pense qu'il faudrait améliorer un peu la description.

Faut te mettre à la place d'une personne qui ne sait pas du tout ce que c'est (doctorant). La propriété peut aussi ne pas exister sur le cluster OAR cible.

Reply:
comme ceci?
oar_mem_core_property_name : str
        The OAR property name corresponding to the memory per core of your cluster, e.g., memcore, mem_core.. If None, warning to users.